src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/DeliverySessionDTO.kt

```diff
@@ -34,6 +34,8 @@ data class UpdateAppointmentDTO(
   val sessionType: AppointmentSessionType? = null,
   val appointmentDeliveryAddress: AddressDTO? = null,
   val npsOfficeCode: String? = null,
+  val appointmentAttendance: UpdateAppointmentAttendanceDTO? = null,
```

Comment:
Nothing is done with these classes

Reply:
These are populated as part of the request 

```
  @Deprecated("superseded by scheduleNewDeliverySessionAppointment and rescheduleDeliverySessionAppointment")
  @PatchMapping("/action-plan/{id}/appointment/{sessionNumber}")
  fun updateSessionAppointment(

```
They are then used in the call to updateSession
```
      updateAppointmentDTO.appointmentAttendance?.attended,
      updateAppointmentDTO.appointmentAttendance?.additionalAttendanceInformation,
      updateAppointmentDTO.appointmentBehaviour?.notifyProbationPractitioner,
      updateAppointmentDTO.appointmentBehaviour?.behaviourDescription,
```